### PR TITLE
Node 10 compel

### PR DESCRIPTION
**Pull request by 13protons**
Ask for node 10 in our engines, travis, and nvm configs. Please help by testing this on your machine too. 

## What does this PR do?
Upgrade requested node versions to >= 10

### Documentation

> Node, and the version we require is not mentioned in our docs :(

 - [ ] Update Spark Docs React
 - [ ] Update Spark Docs Angular
 - [ ] Update Spark Docs Vanilla
 - [ ] Update Component Sass Var/Class Modifier table

### Code
 - [ ] Build Component in Spark Vanilla (Sass, HTML, JS)
 - [ ] Build Component in Spark Angular
 - [ ] Build Component in Spark React
 - [ ] Unit Testing in Spark Vanilla with `npm run test` (100% coverage, 100% passing)
 - [ ] Unit Testing in Spark Angular with `gulp test-angular` (100% coverage, 100% passing)
 - [ ] Unit Testing in Spark React with `gulp test-react` (100% coverage, 100% passing)

### Browser Testing (current version and 1 prior)

> No client side code changed

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v8.10.0
+v10.0.0
```
Comment (yodasw16) [inline .nvmrc]: At one point, this was set to `lts`. Is there a reason not to do that? Current LTS is v10. I know you can use that shorthand in nvmrc. Not sure about the other configs.

**Comment by afebbraro** [comment]
@13protons , we should update the npm version in the pkg json too.

File: .nvmrc (hunk 1 above)
Comment (13protons) [reply .nvmrc]: LTS sounds nice, but it's not very deterministic. What if this blows up our build in November (when node 12 becomes LTS) and we need to react vs. planning the work?

**Comment by afebbraro** [comment]
@13protons , I ended up having to add this to my angular 8 PR for netlify and travis ETC. So, I'm going to close this one since it's in there and we are looking to merge that one in tomorrow.

**Comment by afebbraro** [comment]
This is the PR i mentioned https://github.com/sparkdesignsystem/spark-design-system/pull/1677

**Pull request closed without merging**